guides/common/modules/proc_installing-insights-iop-on-a-connected-project-context-server.adoc

```diff
@@ -20,7 +20,7 @@ endif::[]
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-$ podman login registry.redhat.io
+$ podman login --authfile /etc/foreman/registry-auth.json registry.redhat.io
```

Comment:
The command suggests it runs as a normal user but I think this should be run as root